packages/web-components/src/text/text.styles.ts

```diff
@@ -0,0 +1,103 @@
+import { css } from "@microsoft/fast-element";
+import { fontFamilyBase, fontFamilyMonospace, fontFamilyNumeric, fontSizeBase100, fontSizeBase200, fontSizeBase300, fontSizeBase400, fontSizeBase500, fontSizeBase600, fontSizeHero1000, fontSizeHero700, fontSizeHero800, fontSizeHero900, fontWeightMedium, fontWeightRegular, fontWeightSemibold, lineHeightBase100, lineHeightBase200, lineHeightBase300, lineHeightBase400, lineHeightBase500, lineHeightBase600, lineHeightHero1000, lineHeightHero700, lineHeightHero800, lineHeightHero900 } from "../theme/design-tokens.js";
+
+/** Text styles
+ * @public
+ */
+export const styles = css`
+:host {
+  display: contents;
+}
+::slotted(*) {
+  font-family: ${fontFamilyBase};
+  font-size: ${fontSizeBase300};
+  line-height: ${lineHeightBase300};
+  font-weight: ${fontWeightRegular};
+  text-align: start;
+  display: inline;
+  white-space: normal;
+  overflow: visible;
+  text-overflow: clip;
+}
+:host([truncate]) ::slotted(*),
```

Comment:
FUIR9 does not add nowrap styles when `truncate` is set. It is users responsibility to set `wrap={false}` together with `truncate`:
https://github.com/microsoft/fluentui/blob/9f8fe44d75df0294762fa6f949126e501b6f79a8/packages/react-components/react-text/src/components/Text/useTextStyles.ts#L25-L31
We should unify.